Add news post consideration bullet point to `Official tournament support` wiki article (#10807)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

- A dedicated [screening process](#tournament-screening) to prevent users with serious integrity infringements (cheating, multi-accounting, tournament bans) from playing.
- [Profile badge prizes](#profile-badges) for first-place winners.

> - Potential consideration for a [main-menu banner advertisement](#requesting-in-game-banner-and-news-post-support) during major matches

Reviewer: ```suggestion
- Potential consideration for a [main-menu banner advertisement](#requesting-in-game-banner-and-news-post-support) during major matches.
```

> - Potential consideration for a registration [news post advertisement](#requesting-in-game-banner-and-news-post-support), and a separate news post recapping the final results

Reviewer: "recapping" sounds a bit informal

```suggestion
- Potential consideration for [news coverage](#requesting-in-game-banner-and-news-post-support), one post for advertising registrations and one for summarising final results.
```
